Makefile

```diff
@@ -483,6 +484,26 @@ test-mssql-migration: migrations.mssql.test migrations.individual.mssql.test gen
 	GITEA_ROOT="$(CURDIR)" GITEA_CONF=integrations/mssql.ini ./migrations.mssql.test -test.failfast
 	GITEA_ROOT="$(CURDIR)" GITEA_CONF=integrations/mssql.ini ./migrations.individual.mssql.test -test.failfast
 
+.PHONY: test-e2e
+test-e2e: test-e2e-pgsql
```

Comment:
We should not depend on a specific database for the "offline" target, just assume the user has configured gitea appropriately. If you want psql on the CI, that's fine but it should be a separate target.

Reply:
I guess I'm not sure how how you mean. I updated the makefile to be more like the integration tests, so you can specify what DB type you want to use.

We need to pick a default database to write to for testing. If we use the developer's settings from `app.ini` I think we risk corrupting their development DB. I could create an `e2e.ini.tmpl` file and load that like we do for integration tests.